doc/nrf/ug_nrf52.rst

```diff
@@ -131,6 +138,41 @@ The nRF52 Series devices support running another protocol in parallel with the n
 The :ref:`Multi-Protocol Service Layer (MPSL) <nrfxlib:mpsl>` library provides services for multi-protocol applications.
 
 
+.. |note| replace:: There is currently no support for upgrading the bootloader (:doc:`mcuboot:index`) on nRF52 Series devices.
+
+.. fota_upgrades_start
+
+FOTA upgrades
+*************
+
+|fota_upgrades_def|
+FOTA upgrades can be used to replace the application.
+
+.. note::
+   |note|
+
+To perform a FOTA upgrade, complete the following steps:
+
+1. Make sure that your application supports FOTA upgrades.
+      To download and apply FOTA upgrades, the following requirements apply:
+
+      * You must enable :option:`CONFIG_MCUMGR_SMP_BT` in your application.
+      * |fota_upgrades_req_mcuboot|
+
+#. Create a binary file that contains the new image.
+      |fota_upgrades_building|
+      The :file:`app_update.bin` file is the file that must be downloaded to the device.
+
+#. Download the new image to a device.
+      Use `nRF Connect for Mobile`_ or `nRF Toolbox`_ to upgrade your device with the new firmware.
+
+      To do so, make sure that you can access the :file:`app_update.bin` file from your phone or tablet.
+      Then connect to the device with the mobile app and initiate the DFU process to transfer :file:`app_update.bin` to the device.
+
+      .. note::
+         There is currently no support for the FOTA process in nRF Connect for Desktop.
```

Comment:
not such FOTA but, nrf desktop has their own FOTA solution i believe. @padu can give some more info.

Reply:
This is referring to nRF Connect for Desktop, not to nRF Desktop. ;)

Reply:
ok, understood